ui/src/pages/frontsite/components/Identifiers/Identifiers.js

```diff
@@ -31,7 +31,7 @@ export const IdentifierRows = ({ includeSchemes, identifiers }) => {
     // Only include whitelisted schemes if includeSchemes is set
     if (!includeSchemes || includeSchemes.includes(id.scheme)) {
       const value = { value: id.value, material: id.material };
-      if (id.scheme in identifiers) {
+      if (id.scheme in idsByScheme) {
```

Comment:
are the `alternative_identifiers` displayed correctly in the frontsite in the end?

Reply:
yes